cluster/images/kubernetes-discovery/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Copyright 2016 The Kubernetes Authors.
```

Comment:
A bit confusing but in the diff I sent you, all files in cluster/images/kube-discovery are actually being deleted. Can you drop this directory?